ui/src/app/shared/utils.ts

```diff
@@ -105,5 +105,16 @@ export const Utils = {
     // return a namespace, never return null/undefined, defaults to "default"
     getNamespace(namespace: string) {
         return this.managedNamespace || namespace || this.currentNamespace || 'default';
+    },
+
+    // renders a template literal with context variables
+    renderTemplate(template: string, context: any) {
```

Comment:
minor - I think this can have it's own file - that way you could add a test to it if you liked

Reply:
Thanks! rebased and moved to `links.ts` instead. Also added some tests.